### PR TITLE
Fix compound calculator math and complete tax calculator

### PR DESCRIPTION
**Pull request by Er1ckkkk**

### CODE REVIEW

File: src/com/paynemiller/ch4/TaxCalculator.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+//exercise 14
+public class TaxCalculator {
+    Scanner keyboard = new Scanner(System.in);
+    double orderAmount;
+    String stateInitials;
+    double subtotal;
+    double tax;
+    double total;
+//primitive data types / abstract data types
```
Comment (paynemiller92) [inline src/com/paynemiller/ch4/TaxCalculator.java:13]: Let's remove these comments. They send more like notes than comments about the code!

File: src/com/paynemiller/Runner.java [hunk 2]
```diff
@@ -72,7 +76,7 @@ private static void startCharacterCounter() {
         try {
             counter.countCharacters(word);
         } catch (EmptyWordException e) {
-            e.printStackTrace(); // TODO: Gracefully handle exception.
```
Comment (paynemiller92) [inline src/com/paynemiller/Runner.java:75]: Are we still gonna handle the exception gracefully? If not, let's remote the comment.

File: src/com/paynemiller/ch4/TaxCalculator.java (hunk 1 above)
Comment (Er1ckkkk) [reply src/com/paynemiller/ch4/TaxCalculator.java:13]: Removed as requested.

File: src/com/paynemiller/Runner.java (hunk 2 above)
Comment (Er1ckkkk) [reply src/com/paynemiller/Runner.java:75]: The comment has been removed.

**Pull request merged**